### PR TITLE
tests: fix incompatibility with pytest>=2.8.0

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* FIX Removes calls to PluginManager
  consider_setuptools_entrypoints() removed in PyTest 2.8.0.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -63,7 +63,11 @@ before_script:
   - "inveniomanage database create --quiet || echo ':('"
 
 script:
+<<<<<<< HEAD
```
Comment (jirikuncar) [inline .travis.yml]: fix merge conflicts

File: .travis.yml [hunk 2]
```diff
@@ -48,7 +48,7 @@ python:
 
 before_install:
   - "travis_retry pip install --upgrade pip"
-  - "travis_retry pip install mock twine wheel check-manifest"
+  - "travis_retry pip install check-manifest mock twine wheel"
```
Comment (jirikuncar) [inline .travis.yml]: add `coveralls`

**Comment by jirikuncar** [comment]
- [ ] include url_converter in `EXTENSIONS` in `.travis.invenio.cfg`

File: .travis.invenio.cfg [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+CFG_BIBSCHED_PROCESS_USER = getpass.getuser()
+
+DEBUG = False
+SECRET_KEY = 'MY_SECRET'
+
+# Disable all automatic asset building - false is /usr/bin/false.
+ASSETS_AUTO_BUILD = False
+
+PACKAGES = [
+    'invenio_testing',
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: remove it

File: .travis.invenio.cfg [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    'invenio_base',
+]
+
+EXTENSIONS = [
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: see https://github.com/jirikuncar/invenio-search/commit/58f9c0b65b7207eaebc22d1eccab0f1ddb44cd2b

File: .travis.invenio.cfg [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    'url_converter',
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: https://github.com/inveniosoftware/invenio-deposit/blob/master/invenio_deposit/url_converters.py#L72

**Comment by tiborsimko** [comment]
@SamiHiltunen Can you please rebase against latest `master` now that PR #24 was merged?

**Comment by SamiHiltunen** [comment]
@tiborsimko rebased

File: .travis.invenio.cfg [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    'invenio_records',
+    'invenio_oauth2server'
+    'invenio_knowledge',
+    'invenio_access',
+    'invenio_pidstore',
+    'invenio_formatter',
+    'invenio_base',
+]
+
+EXTENSIONS = ['invenio_deposit.url_converters']
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: please include also `invenio_base.config:EXTENSIONS`

**Comment by SamiHiltunen** [comment]
If you need this for old release, please ping me.

**Pull request closed without merging**